test/TestProjectWithSDKRef/TestProjectWithSDKRef.csproj

```diff
@@ -9,6 +9,8 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="TestProject" Version="#{NBGV_NuGetPackageVersion}#" />
+        <PackageReference Include="TestProject" Condition="'$(TestWildcard)'=='Double'" Version="1.*-*" />
```

Comment:
Now that I see this I'm thinking, can't we specify the Version we want to use entirely from the command line and test our various scenario's that way? So we just put `Version="$(DependencyVersion)"` in here (with a single `PackageReference` element again) and then control the `DependencyVersion` property from the command line. That would make this a lot cleaner imho, but I am sorry for changing my mind about this every time I see it ;).